packages/common/src/index.ts

```diff
@@ -196,6 +197,29 @@ export default class Common {
     return value
   }
 
+  /**
+   * Returns a parameter corresponding to an EIP
+   * @param topic Parameter topic ('gasConfig', 'gasPrices', 'vm', 'pow')
+   * @param name Parameter name (e.g. 'minGasLimit' for 'gasConfig' topic)
+   * @param hardfork Name of the EIP (e.g. 'EIP2537')
```

Comment:
The parameter here should be `eip`, not `hardfork`

Reply:
Done (see on next push)